examples/nlp/distributed_training_with_keras_nlp.py

```diff
@@ -0,0 +1,124 @@
+
+"""
+Title: Distributed Training with KerasNLP
+Author: Anshuman Mishra
+Date created: 2023/06/04
+Last modified: 2023/06/05
+Description: This tutorial demonstrates how to perform distributed training using KerasNLP. 
+It covers the steps for training a BERT-based masked language model on the wikitext-2 
+dataset, using multiple GPUs for accelerated training.
+
+Accelerator: GPU
```

Comment:
Also we assume our default accelerator to be GPU here rather than setting up Logical Devices from CPU